cpp/src/arrow/compute/kernels/vector_sort.cc

```diff
@@ -2340,6 +2340,9 @@ class SelectKUnstableMetaFunction : public MetaFunction {
       return Status::Invalid("SelectK requires a nonnegative `k`, got ",
```

Comment:
Can you change the error messages to use the registered name of the function ("select_k_unstable")?